src/user-managerment/module/pages/Dashboard/Home/BuyCoin.js

```diff
@@ -31,6 +38,7 @@ const BuyCoin = () => {
 
     // STATES
     const [assetAccounts, setAssetAccounts] = useState([])
+    const [singleListing, setSingleListing] = useState(null)
```

Comment:
isnt it better this is an empty object than null, Just asking out of curiosity @michael7666

Reply:
@monsele You can either pass null or an empty object, it dependent on what you are using it for sha